pytaser/generator.py

```diff
@@ -681,7 +679,7 @@ def generate_tas(self, energy_min, energy_max, gaussian_width=0.2, step=0.1):
         )
 
     @classmethod
-    def from_mpid(cls, api_key = None, mpid, temperature, concentration, bg=None):
+    def from_mpid(cls, mpid, temperature, concentration, bg=None, api_key=None):
```

Comment:
keyword arguments have to be after all positional arguments - this code wouldn't run